telebot/types.py

```diff
@@ -12647,7 +12647,7 @@ class InputChecklist(JsonSerializable):
     """
     def __init__(self, title: str,parse_mode: Optional[str] = None,
                     title_entities: Optional[List[MessageEntity]] = None,
-                    tasks: List[InputChecklistTask], others_can_add_tasks: Optional[bool] = None,
+                    tasks: List[InputChecklistTask]=None, others_can_add_tasks: Optional[bool] = None,
```

Comment:
Sorry, I was wrong here. 

tasks are not optional, so should be without "None" and goes before title_entities and parse mode...

You were correct. Sorry for that, I missed it's not optional.